src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1352,6 +1353,44 @@ public ItemStack get(PlayerArmorChangeEvent e) {
 				}
 			}, 0);
 		}
+		//PlayerInventorySlotChangeEvent
+		if (Skript.classExists("io.papermc.paper.event.player.PlayerInventorySlotChangeEvent")) {
+			EventValues.registerEventValue(PlayerInventorySlotChangeEvent.class, ItemStack.class, new Getter<ItemStack, PlayerInventorySlotChangeEvent>() {
+				@Override
+				@Nullable
+				public ItemStack get(PlayerInventorySlotChangeEvent event) {
+					return event.getNewItemStack();
+				}
+			}, EventValues.TIME_NOW);
+			EventValues.registerEventValue(PlayerInventorySlotChangeEvent.class, ItemStack.class, new Getter<ItemStack, PlayerInventorySlotChangeEvent>() {
+				@Override
+				@Nullable
+				public ItemStack get(PlayerInventorySlotChangeEvent event) {
+					return event.getOldItemStack();
+				}
+			}, EventValues.TIME_PAST);
+			EventValues.registerEventValue(PlayerInventorySlotChangeEvent.class, ItemStack.class, new Getter<ItemStack, PlayerInventorySlotChangeEvent>() {
+				@Override
+				@Nullable
+				public ItemStack get(PlayerInventorySlotChangeEvent event) {
+					return event.getNewItemStack();
+				}
+			}, EventValues.TIME_FUTURE);
```

Comment:
No need to register a future event value if it matches the default event value, since if a future/past event value is used and it isn't registered, it'll default to the now event value

Reply:
oh interesting, didn't know that happened!